archunit/src/test/java/com/tngtech/archunit/library/testclasses/packages/correct/twoimplementationsonetestdir1/OnlyOneImplementationHaveTestAndItIsMatchingImplPackageTest.java

```diff
@@ -0,0 +1,4 @@
+package com.tngtech.archunit.library.testclasses.packages.correct.twoimplementationsonetestdir1;
+
+public class OnlyOneImplementationHaveTestAndItIsMatchingImplPackageTest {
```

Comment:
For your test to cover the use case of #1367, this should be
```suggestion
class MultipleImplementationsWithOnlyOneTestAndInRightPackageTest {
```
to make your new test actually fail with the old implementation. 🙂

In addition: I'd drop the `AndInRightPackage` part from this name: It's a relevant aspect to have classes with the same simple name in different packages, so: _Is it right or is it wrong??_ 
How about `SimpleNameThatOccursInSeveralPackages`?

Reply:
I was reneming classes at the end and forgot about that one 🤦 Thanks for this suggestion - applied!